ui/js/models/repository.js

```diff
@@ -37,9 +37,9 @@ treeherder.factory('ThRepositoryModel', [
                   const groups = $rootScope.repos.reduce((acc, repo, idx, arr, group = repo => repo.repository_group.name) => (
                       { ...acc, [group(repo)]: [...acc[group(repo)] || [], repo] }
                   ), {});
-                  _.each(groups, function (reposAr, gName) {
-                      orderedRepoGroups[thRepoGroupOrder[gName] || gName] = { name: gName, repos: reposAr };
-                  });
+                Object.keys(groups).forEach((key) => {
```

Comment:
In this case ``Object.entries`` would be better so you don't have to use the ``groups[key]``.  It's a bit of a nit-pick, but these functions are very iterative, so I think it might be slightly more performant.